inferno-core/CHANGELOG.md

```diff
@@ -1,5 +1,9 @@
 # Revision History for inferno-core
 
+## 0.1.2 -- 2023-02-09
```

Comment:
@siddharth-krishna can we start using this convention? [semantic versioning](https://semver.org/)

Reply:
Sure!